vizro-core/docs/pages/user-guides/dynamic-data.md

```diff
@@ -0,0 +1,134 @@
+# Dynamic data
+
+A dynamic data source is a Python function that returns a pandas DataFrame. This function is executed when the dashboard is initially started and _can be executed again while the dashboard is running_. This makes it possible to refresh the data shown in your dashboard without restarting the dashboard itself. If you do not require this functionality then you should use [static data](static-data.md) instead.
+
+Unlike static data, dynamic data cannot be supplied directly into the `data_frame` argument of a `figure`. Instead, it must first be added to the Data Manager and then referenced by name.
+
+The below toy example demonstrates how dynamic data is updated every time the page is refreshed. When you run the code and refresh the page the data shown will change every time because the function `load_iris_data` is re-run. The example uses the Iris dataset saved to a file `iris.csv` in the same directory as `app.py`. This data can be generated using `px.data.iris()` or [downloaded](../../assets/user_guides/data/iris.csv).
```

Comment:
```suggestion
The example below shows how dynamic data is updated every time the page is refreshed. When you run the code and refresh the page the data shown will change every time because the function `load_iris_data` is re-run. The example uses the Iris dataset saved to a file `iris.csv` in the same directory as `app.py`. This data can be generated using `px.data.iris()` or [downloaded](../../assets/user_guides/data/iris.csv).
```

Reply:
> The below toy example demonstrates how dynamic data is updated every time the page is refreshed. When you run the code and refresh the page the data shown will change every time because the function `load_iris_data` is re-run. The example uses the Iris dataset saved to a file `iris.csv` in the same directory as `app.py`. This data can be generated using `px.data.iris()` or [downloaded](../../assets/user_guides/data/iris.csv).

I think we need to rephrase this a bit as the suggestion is that if you refresh the page you change the data. In most realtime systems, I'd expect the opposite (the data changes, the page refreshes with the update) but I guess we aren't claiming that here. So what is happening is that the page is updated the data is fetched again, right? Could we try this suggestion...?

Reply:
```suggestion
The example below shows how data is fetched dynamically every time the page is refreshed. When you run the code and refresh the page the data is fetched by the function `load_iris_data` is re-run. The example uses the Iris dataset saved to a file `iris.csv` in the same directory as `app.py`. This data can be generated using `px.data.iris()` or [downloaded](../../assets/user_guides/data/iris.csv).

```

Reply:
I know it's subtle but it removes (to my mind) any chance people get confused that a change in the data causes an update of the UI. Which is not what is happening AFAICT.

Reply:
You are absolutely right, thank you for the suggestion! I will make the change.